packages/dashboard/src/components/DashboardFilters/DashboardFiltersEditor/components/APIModal.tsx

```diff
@@ -0,0 +1,127 @@
+import { useState } from 'react'
+import { SharedFilter } from '../../../../types/SharedFilter'
+import Tooltip from '@cdc/core/components/ui/Tooltip'
+import Icon from '@cdc/core/components/ui/Icon'
+
+type APIModalProps = {
+  filter: SharedFilter
+  isSubgroup: boolean
+  isNestedDropdown: boolean
+  textSelector: string
+  valueSelector: string
+  updateAPIFilter: Function
+}
+
+const APIModal: React.FC<APIModalProps> = ({
+  filter,
+  isNestedDropdown,
+  updateAPIFilter,
+  valueSelector,
+  textSelector
+}) => {
+  const [APIEndpoint, setAPIEndpoint] = useState(filter.apiFilter?.apiEndpoint || '')
+  const [APIValueSelector, setAPIValueSelector] = useState(filter.apiFilter?.valueSelector || '')
+  const [APITextSelector, setAPITextSelector] = useState(filter.apiFilter?.textSelector || '')
+  const [APISubGroupValueSelector, setAPISubGroupValueSelector] = useState(
+    filter.apiFilter?.subgroupValueSelector || ''
+  )
+  const [APISubGroupTextSelector, setAPISubGroupTextSelector] = useState(filter.apiFilter?.subgroupTextSelector || '')
+  return (
+    <fieldset className='edit-block mb-1 px-3 cdc-open-viz-module'>
+      <label className='d-block'>
+        <span>API Endpoint: </span>
+        <textarea
+          value={APIEndpoint}
+          rows={4}
+          onChange={e => setAPIEndpoint(e.target.value)}
+          className='w-100'
+          style={{ minHeight: '1.5rem', maxWidth: '90%' }}
+        />
+        {isNestedDropdown && (
+          <Tooltip style={{ textTransform: 'none' }}>
+            <Tooltip.Target>
+              <Icon display='question' style={{ marginLeft: '0.5rem' }} />
+            </Tooltip.Target>
+            <Tooltip.Content>
+              <p>Your API Endpoint should return both value selector values.</p>
+            </Tooltip.Content>
+          </Tooltip>
+        )}
+      </label>
+      <div className='d-flex'>
+        <div className={`w-50${isNestedDropdown ? ' border border-dark p-1 m-1' : ''}`}>
+          <label>
+            <span>Value Selector: </span>
+            <input value={APIValueSelector || ''} onChange={e => setAPIValueSelector(e.target.value)} />
+            <Tooltip style={{ textTransform: 'none' }}>
+              <Tooltip.Target>
+                <Icon display='question' style={{ marginLeft: '0.5rem' }} />
+              </Tooltip.Target>
+              <Tooltip.Content>
+                <p>Value to use in the html option element</p>
+              </Tooltip.Content>
+            </Tooltip>
+            <div>{` * Required`}</div>
```

Comment:
can you wrap this in either a span or a p tag to indicated that the inner html is text? I think it's better for accessibility.